rules/inform-about-content-deletion/rule.md

```diff
@@ -0,0 +1,70 @@
+---
+type: rule
+title: Do you inform others when deleting or making fundamental content changes?
+uri: inform-about-content-deletion
+authors:
+  - title: Adam Cogan
+    url: https://www.ssw.com.au/people/adam-cogan
+related:
+  - keep-email-history
+  - change-from-x-to-y
+  - do-you-know-zz-ed-files-must-not-exist-in-source-control
+  - comments-do-you-enforce-comments-with-check-ins
+created: 2023-08-08T22:48:46.280Z
+guid: cfff85d6-1cd9-4872-a29f-ab68cf90a113
+
+---
+
+When it comes to making significant alterations or deletions to someone else's content, it's a good practice to inform them and document it, by sending an ["as per our conversation email"](/as-per-our-conversation-emails) for example.
+
+Clear communication helps maintain transparency and respect for the original author's work.
+            
+<!--endintro-->
+
+This is especially important in collaborative environments, where changes can impact the overall project or its documentation. Keeping others informed allows for discussions or adjustments if needed.
+
+Understanding the history of content is essential for context, accuracy, and informed decision-making.
+
+Being diligent about content deletion not only respects contributions but also mitigates risks and fosters a culture of understanding and problem-solving. 
+
+::: email-template  
+|          |     |
+| -------- | --- |
+| To:      | Dave |
+| Cc:      | John, Danny |
+| Subject: | FAQ - Removed irrelevant section  |  
+::: email-content  
+
+### Hi Dave,  
+
+As per our conversation, I have removed the section below from the FAQs since it is redundant, as you no longer join Daily Scrums. 
+
+I have Cc'ed others who have edited this page.
+
+From
+
+### FAQ
+
+<p style="background-color:#ff0000;color:#fff;font-weight:bolder;"> 
+Q: What if your team already does Daily Scrums?  
+</p>
+
+<p style="background-color:#ff0000;color:#fff;font-weight:bolder;"> 
+A: Prior to the Daily Scrum, ping Dave and say "Starting our Daily Scrum in a minute". Do your Daily Scrum as normal and add in Dave/Product Owner. 
+After the call, one person (called the Scribe) should send an email to the whole team. 
+:</p>
+
+**Q: What if I only have one day until I go on client work?**  
+...
+
+To
+
+### FAQ
+**Q: What if I only have one day until I go on client work?**  
+...
+
+:::  
+:::  
+::: good  
+Figure: Good example - Being careful and documenting content deletion 
```

Comment:
Should we say **considerate** instead of **careful**? It's the same for me in this use case with a little extra EQ 🙂

Reply:
```suggestion
Figure: Good example - Being considerate and documenting content deletion 
```